docs/knowledge-base/monitoring/Prometheus.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Prometheus
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Prometheus
+You can access Prometheus at https://prometheus.mapcolonies.net.
+
+Prometheus is the core system for **collecting and storing metrics** as time-series data, meaning each metric is stored with the timestamp of when it was recorded, alongside labels that provide additional context to the metric.
+
+## Home Screen Overview 
+The Prometheus home screen provides an overview of the current status of the Prometheus server. Here, you can see the targets that are being monitored, the current configuration of Prometheus, and access the built-in query interface.
+![prometheus homescreen](./img/prometheus-homescreen.png)
+
+## Prometheus Scrape Configuration
+Prometheus scrapes metrics from pods located in specific namespaces and annotated correctly. Below is an example configuration snippet:
+
+  ```yaml
+  {
+    - job_name: k8s-scrape
+    kubernetes_sd_configs:
+      - role: pod
+        namespaces:
+          own_namespace: true
+          names:
+            - projectA
+            - projectB
+    relabel_configs:
+      - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+        action: keep
+        regex: true
+  }
+  ```
```

Comment:
here add sub-headline that indicates that this section is the other side - the developers who want to connect to this service.

Reply:
done